src/api/cmd/chart-repo-sync/sync.go

```diff
@@ -68,21 +75,22 @@ func main() {
 		log.WithFields(log.Fields{"host": *dbURL}).Fatal(err)
 	}
 
-	if err := sync(flag.Arg(0), flag.Arg(1)); err != nil {
+	if err := syncRepo(flag.Arg(0), flag.Arg(1)); err != nil {
 		log.WithError(err).Error("sync failed")
 		os.Exit(1)
 	}
 }
 
-// Syncing is performed in the following sequential steps:
+// Syncing is performed in the following steps:
 // 1. Update database to match chart metadata from index
-// 3. Process icons for charts
-// 4. Process READMEs for each chart version
+// 3. Concurrently process icons for charts (concurrently)
```

Comment:
I do not fully understand the numbering here.

Reply:
Apparently I don't know how to count, will fix.

Reply:
I understand you, I only know how to count in roman numbers :trollface: